API/Data/Repositories/ReadingListRepository.cs

```diff
@@ -241,8 +242,11 @@ public async Task<IEnumerable<ReadingListItemDto>> AddReadingProgressModifiers(i
 
         foreach (var item in items)
         {
-            var progress = userProgress.Where(p => p.ChapterId == item.ChapterId);
-            item.PagesRead = progress.Sum(p => p.PagesRead);
+            var progress = userProgress.SingleOrDefault(p => p.ChapterId == item.ChapterId);
+            if (progress == null) continue;
+
+            item.PagesRead = progress.PagesRead;
```

Comment:
Why are you removing the Sum?

Reply:
Because I thought a user can't have multiple reading status for the same chapter and I also needed to read the date. This is not the case?

Reply:
This is true, but there are some old versions that had a bug and may have multiple records, hence why it's done with the sum.

Reply:
I do not see how the sum can solve this problem since you will have a page number which can exceed the number of pages of the chapter. I'm thinking of using the maximum page number instead. I can't sum dates!

Reply:
I think the correct solution would be to remove the duplicates from the database and add a uniqueness constraint.

Reply:
Your solution is outside the scope of the PR. You don't need to sum dates, you can just use .Max() to get the latest date. Let's not change any extra code unless it's necessary.

Reply:
ok